src/main/resources/application.yml

```diff
@@ -101,4 +101,7 @@ notification:
     reservation: "care_reservation"
     update_reservation: "care_update"
     accepted_reservation: "care_accepted"
-    rejected_reservation: "care_rejected"
\ No newline at end of file
+    rejected_reservation: "care_rejected"
+
+care:
+  accept-time-window: P0D        # 0 days, meaning the care must be accepted the day it has started
```

Comment:
moze dajmy czas do nastepnego dnia, wtedy opiekun bedzie mial czas zaznaczyc w dzien opieki plus 1 dzien do polnocy.
Wgl mam watpliwosci czy powinnismy robic outdated jesli opiekun nie zaznaczy, bo opiekun moze nie pilnowac tego statusu. Z drugiej strony jak na allegro ktos kupi twoj produkt to musisz pamietac zeby go wyslac xD wiec moze i git

Reply:
> moze dajmy czas do nastepnego dnia, wtedy opiekun bedzie mial czas zaznaczyc w dzien opieki plus 1 dzien do polnocy.

Najlepiej jakby to było oznaczane od razu przy odbiorze zwierzęcia. Imo to musi być tego samego dnia bo to potwierdzenie że usługa została wykonana. Bo tak to jak ktoś zostawi zwierze na dzień i klientowi coś się nie będzie podobało, to opiekun nie zaznaczy jako wykonane żeby opinii złej nie dostać

Reply:
Wg kodu ready to proceed to też status gdzie można dodać opinie

Reply:
A chodzi o to że opiekun nie zawsze będzie chciał się logować wchodzić na stronę itp i jak nie zdąży to mimo że jest zwierzę to będzie outdated